cmd/fyne/internal/commands/deprecated.go

```diff
@@ -0,0 +1,44 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime/debug"
+
+	"github.com/urfave/cli/v2"
+)
+
+// Vendor returns the vendor cli command.
+//
+// Deprecated: Use "go mod vendor" instead.
+func Vendor() *cli.Command {
+	return &cli.Command{
+		Name:  "vendor",
+		Usage: "Deprecated: Use \"go mod vendor\" instead.",
+		Action: func(_ *cli.Context) error {
+			cmd := exec.Command("go", "mod", "vendor")
+			cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr
+			return cmd.Run()
+		},
+	}
+}
+
+// Version returns the cli command for the program version.
+//
+// Deprecated: Use "fyne --version" or "fyne -v" instead.
+func Version() *cli.Command {
+	return &cli.Command{
+		Name:  "version",
+		Usage: "Deprecated: Use \"fyne --version\" or \"fyne -v\" instead.",
```

Comment:
Is there any particular reason to deprecate "fyne version" when it follows the pattern of other subcommands?

Reply:
It is because `--version` (which is built into urfave/cli) basically is the standard way of showing versions on UNIX. It also cleans up our code a slight bit to not add one extra command 🙂

Reply:
But it removes the expected location for the version command for anyone working with go tools...

Reply:
That is a very good point. Didn't think about that. Will see what I can do 🙂

Reply:
Moved back to what we had.